pyproject.toml

```diff
@@ -20,8 +20,8 @@ requires-python = ">=3.6"
 dependencies = [
     "ase",
     "matplotlib",
-    "numpy",
-    "pyopencl",
+    "numpy<=1.21.6",
+#    "pyopencl",
```

Comment:
Why the limit on numpy version and removed dependency on pyopencl?

Reply:
Added a comment on numpy upper limit, moved pyopencl to optional dependencies.